administrator/components/com_joomgallery/helpers/html/joomconfig.php

```diff
@@ -129,4 +129,353 @@ public static function row($key, $type, $name, $info, $display = true, $attribs
+                          <p><strong <?php echo JHTML::_('joomgallery.tip', JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_DIMENSION_TXT'), JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_DIMENSION_LBL'), true); ?>><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_DIMENSION_LBL');?>:</strong> <?php echo $info->thumbR->dimension;?></p>
+                          <p><strong <?php echo JHTML::_('joomgallery.tip', JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_FRAMES_TXT'), JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_FRAMES_LBL'), true); ?>><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_FRAMES_LBL');?>:</strong> <?php echo $info->thumbR->frames;?></p>
+                          <p><strong <?php echo JHTML::_('joomgallery.tip', JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_GENTIME_TXT'), JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_GENTIME_LBL'), true); ?>><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_GENTIME_LBL');?>:</strong> <?php echo round($info->thumbR->processing_time,3);?> sec</p>
+                          <p><strong <?php echo JHTML::_('joomgallery.tip', JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_GENMEMORY_TXT'), JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_GENMEMORY_LBL'), true); ?>><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_GENMEMORY_LBL');?>:</strong> <?php echo round(($info->thumbR->used_memory) / pow(1000,2),2);?> MB</p>
+                        <?php else: ?>
+                          <p><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_IMGMISSING'); ?></p>
+                        <?php endif; ?>
+                      </td>
+                    </tr>
+                    <tr>
+                      <td colspan="3" class="center">
+                      <button id="btnModal_thumb" class="btn btn-primary" onclick="openModal(event)" <?php if($thumb_exist) {echo '';} else {echo 'disabled';}?>><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_COMPAREBOX_BTN'); ?></button>
+                      </td>
+                    </tr>
+                  </tbody>
+                </table>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <?php if($detail_exist || $thumb_exist): ?>
+          <div class="modal hide fade jviewport-width80 in" id="modalCompare" tabindex="-1" aria-labelledby="modalCompare_title" aria-hidden="true">
+            <div class="modal-header">
+              <button type="button" class="close novalidate" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+              </button>
+              <h3 id="modalCompare_title" thumb_img="<?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_COMPAREBOX_THUMBTITLE'); ?>" detail_img="<?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_COMPAREBOX_DETAILTITLE'); ?>">modal_title</h3>
+            </div>
```

Comment:
Add
```
              <div class="center">
                <input id="modalImg_width" type="number" min="1" value="400">
                <span> px </span>
                <button id="modalImg_btn" class="btn btn-primary" onclick="setIMGwidth(event);"><?php echo JText::_('COM_JOOMGALLERY_CONFIG_GS_IP_TEST_COMPAREBOX_APPLY'); ?></button>
              </div>
```
in modal header and remove modal-footer
to always be able to change the value of width.